lstchain/visualization/plot_drs4.py

```diff
@@ -36,9 +36,21 @@ def plot_pedestals(data_file, pedestal_file, run=0 , plot_file="none", tel_id=1,
      offset_value: baseline off_set
      """
 
-    # r0 calibrator
-    r0_calib = LSTR0Corrections(pedestal_path=pedestal_file, offset=offset_value,
-                                tel_id=tel_id )
+    # plot open pdf
+    if plot_file != "none":
+        pp = PdfPages(plot_file)
```

Comment:
This might lead to broken pdf files as it is important to properly close a `PdfPages` object. `PdfPages` should always be used with a context manager.